api/base/filters.py

```diff
@@ -400,6 +401,102 @@ def convert_special_params_to_odm_query(self, field_name, query_params, key, dat
         """
         pass
 
+class DjangoFilterMixin(FilterMixin):
```

Comment:
Most of this is copied from the odmfiltermixin, the main things that changed is the _operation_to_query